### PR TITLE
Update template, fix issue

### LINKED ISSUE #44: NACHO (qc) report not published
### Description of the bug

Currently, the `*.html` report that is generated by `NACHO` is not defined as output file since it is commented out here: https://github.com/nf-core/nanostring/blob/5539b33b1821dbd4cf4e68c1053c6f89f8e308fa/modules/local/nacho/qc.nf#L15 

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by christopher-mohr**
This 
- updates the pipeline to TEMPLATE `2.9`
- updates nf-core modules
- fixes #44. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -55,9 +55,8 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 
 The typical command for running the pipeline is as follows:
 
-```bash
-nextflow run nf-core/nanostring --input samplesheet.csv --outdir <OUTDIR> -profile docker
-```
+````bash
+nextflow run nf-core/nanostring --input ./samplesheet.csv --outdir ./results -profile docker```
```
Comment (grst) [inline docs/usage.md:59]:
````suggestion
nextflow run nf-core/nanostring --input ./samplesheet.csv --outdir ./results -profile docker
```
````

File: docs/usage.md [hunk 2]
```diff
@@ -68,14 +67,15 @@ work                # Directory containing the nextflow working files
 <OUTDIR>            # Finished results in specified location (defined with --outdir)
 .nextflow_log       # Log file from Nextflow
 # Other nextflow hidden files, eg. history of pipeline runs and old logs.
-```
+````
```
Comment (grst) [inline docs/usage.md:70]:
````suggestion
```
````

File: docs/usage.md (hunk 1 above)
Comment (grst) [reply docs/usage.md:59]: Something seems off with the code block here

File: docs/usage.md [hunk 3]
```diff
@@ -209,6 +209,24 @@ To use a different container from the default container or conda environment spe
...
+If you have any questions or issues please send us a message on [Slack](https://nf-co.re/join/slack) on the [`#configs` channel](https://nfcore.slack.com/channels/configs).
+
+## Azure Resource Requests
+
+To be used with the `azurebatch` profile by specifying the `-profile azurebatch`.
+We recommend providing a compute `params.vm_type` of `Standard_D16_v3` VMs by default but these options can be changed if required.
+
+Note that the choice of VM size depends on your quota and the overall workload during the analysis.
+For a thorough list, please refer the [Azure Sizes for virtual machines in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes).
+
```
Comment (grst) [inline docs/usage.md:229]: Did you add this on purpose? This doesn't seem to be part of the template README anymore and I'd also argue that this is pretty specific information that might better be linked to.

**Review by grst — APPROVED** [review]
Looks good to me overall, I just have two minor comments on the README.

File: docs/usage.md (hunk 3 above)
Comment (christopher-mohr) [reply docs/usage.md:230]: No, I didn't add it on purpose. Must have slipt in while taking over current changes. I will remove it, thanks!

Comment (christopher-mohr) [reply docs/usage.md:230]: @grst To which part of the docs do you prefer exactly? Looking at the `TEMPLATE` again I can see the `nf-core/configs` and `Azure` part in there.

Comment (christopher-mohr) [reply docs/usage.md:230]: Ah, but this [usage.md](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/docs/usage.md) we are looking at here. :)

Comment (grst) [reply docs/usage.md:230]: Ooop, my bad. All good then!

**Pull request merged**